miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -20,686 +20,696 @@
  *
  */
 
-/*
- * A BoxPosition represents a single position (cell) in the Box visualization (table).
- * It is responsible for handling the functionality of a cell. This includes changing the title and image
- * associated with the position.
- */
-var BoxPosition = function(opts) {
-  var self = {};
-
-  isMultiSelectClick = function(event) {
-    // multi-select is accessible by pressing the Command key (metaKey) on Mac,
-    // or the Control key (ctrlKey) on Windows or Linux
-    return navigator.platform.indexOf('Mac') != -1 ? event.metaKey : event.ctrlKey;
-  };
-
-  self.title = opts.title || '';
-  self.parentVisual = opts.parentVisual;
-  self.row = opts.row;
-  self.col = opts.col;
-  self.cell = opts.cell;
-  self.selectedImg = opts.selectedImg || null;
-  self.unselectedImg = opts.unselectedImg || null;
-  self.element = opts.element || jQuery('<img>');
-
-  self.setTitle = function() {
-    self.cell.prop('title', Box.utils.getPositionString(self.row, self.col) + ': ' + self.title);
-  };
+(function(Box, $, undefined) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Do not use "undefined" to declare a parameter - use another name. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=javascript%3AS2137)